docs/FIRRTLAnnotations.md

```diff
@@ -305,7 +305,7 @@ Example:
 
 | Property   | Type   | Description                             |
 | ---------- | ------ | -------------                           |
-| class      | string | `firrte.transforms.DontTouchAnnotation` |
+| class      | string | `firrtl.transforms.DontTouchAnnotation` |
```

Comment:
In the future, feel free to land these directly on main as separate commits when you see these. 🙏